docs/src/docs.md

```diff
@@ -188,7 +188,7 @@ This example also shows the use of `Float32`, a 2D simulation and no specified u
 atoms = [Atom(mass=1.0f0), Atom(mass=1.0f0)]
 coords = [SVector(0.3f0, 0.5f0), SVector(0.7f0, 0.5f0)]
 velocities = [SVector(0.0f0, 1.0f0), SVector(0.0f0, -1.0f0)]
-pairwise_inters = (Gravity(nl_only=false, G=1.5f0),)
+pairwise_inters = (Gravity(nl_only=false, G=-1.5f0),)
```

Comment:
Since the gravitational constant should be positive, this negative sign should go in line 32 and also line 52 of `src/interactions/gravity.jl`.